ops/op2/dispatch_fast.rs

```diff
@@ -471,6 +473,22 @@ pub(crate) fn generate_dispatch_fast(
     quote!(Self:: #name)
   };
 
+  let with_scope = if generator_state.needs_fast_scope {
+    generator_state.needs_fast_api_callback_options = true;
+    gs_quote!(generator_state(scope, fast_api_callback_options) => (
+      let mut #scope = unsafe {
+        if !this.is_object() {
+          #fast_api_callback_options.fallback = true;
+          // SAFETY: All fast return types have zero as a valid value
+          return unsafe { std::mem::zeroed() };
+        };
+        deno_core::v8::CallbackScope::new(this)
```

Comment:
`Object::unwrap` could take a &mut Isolate to avoid constructing `v8::CallbackScope` which is a expensive

Reply:
Oh and also, once we remove `fallback` the `!this.object()` check will have to throw but to do that we need a scope :)

Reply:
we can open a scope off of `FastApiCallbackOptions.isolate` (the intended way, also lets you use ops without a receiver) but it's not in the version of v8 we're using yet.